Subheaders with social media

Leehman opened the pull request:
Please review the 'subheaders'.

Leehman commented:
![snap 2018-01-29 at 10 50 29](https://user-images.githubusercontent.com/4561068/35519443-4065fbba-04e2-11e8-9102-821a752620c2.png)

Leehman commented:
Image as requested.

johnleider commented on examples/subheaders/Subheader.vue:
```diff
@@ -0,0 +1,79 @@
+          </v-layout>
+          <v-subheader>Places to See </v-subheader>
+          <v-layout row wrap>
+            <v-spacer></v-spacer>
```
Is this needed?

Leehman replied:
if this is in regards to v-spacer, no and its removed.

johnleider commented on examples/subheaders/Subheader.vue:
```diff
@@ -0,0 +1,79 @@
+      </v-card>
+    </v-flex>
+  </v-layout>
+</template>
```
line break between template and script

johnleider commented on examples/subheaders/Subheader.vue:
```diff
@@ -0,0 +1,79 @@
+      }
+    ]
+  })
+}
```
fix indentation of everything in here

johnleider commented on examples/subheaders/Subheader.vue:
```diff
@@ -0,0 +1,79 @@
+            <v-spacer></v-spacer>
+            <v-flex xs4 v-for="card in cards1" :key="card.title">
+              <v-card>
+                <v-card-media height="300px" :src="`https://picsum.photos/200/300?image=586`">
```
If this is not dynamic, does not need to be a bound parameter

Leehman replied:
Fixed as requested.

johnleider commented on examples/subheaders/Subheader.vue:
```diff
@@ -0,0 +1,79 @@
+            <v-flex xs4 v-for="card in cards1" :key="card.title">
+              <v-card>
+                <v-card-media height="300px" :src="`https://picsum.photos/200/300?image=586`">
+                  <span class="headline white--text  pl-3 pt-3" v-text="card.title"></span>
```
extra space after white--text

Leehman replied:
Fixed as requested.

johnleider commented on examples/subheaders/Subheader.vue:
```diff
@@ -0,0 +1,79 @@
+        <v-container fluid grid-list-md class="grey lighten-4">
+          <v-subheader>Server &amp; Ops Management</v-subheader>
+          <v-layout row wrap>
+            <v-spacer></v-spacer>
```
is this needed?

Leehman replied:
if this is in regards to v-spacer, no and its removed.

bdeo commented:
Additionally, it's usually against company guidelines to use logos with modified colors. It might be a good idea to use the official colors for these logos.

johnleider commented:
I agree with @bdeo 's feedback.

Leehman commented:
![snap 2018-01-29 at 14 08 33](https://user-images.githubusercontent.com/4561068/35532057-e8d179c2-0506-11e8-9a1e-ff07d7606e7a.png)

Leehman commented:
Switched ide to Visual Code. Changed icon text to white, Social media icons should reflect colors of brand, code modified as requested per JL, white space removed with ide component.

jacekkarczmarczyk commented on util/helpers.js:
```diff
@@ -32,3 +32,18 @@ export function isOnSale (variants) {
 }
+
+export function randomNumber (min, max) {
+  return Math.floor(Math.random() * max) + min
```
`Math.floor(Math.random() * (max - min + 1)) + min`
